### PR TITLE
Added a way to inspect installation scripts on Powershell(Windows)

### PR DESCRIPTION
**Pull request by FishAlchemist**
### command
 ```powershell
    powershell -c "irm https://astral.sh/uv/install.ps1 | more"
 ```
Add a method to inspect installation script files that works on Windows, it may not work on other platforms supported by PowerShell. <br>
Other platforms information:
[PowerShell differences on non-Windows platforms / General Unix interoperability changes](https://learn.microsoft.com/en-us/powershell/scripting/whats-new/unix-support?view=powershell-7.4#general-unix-interoperability-changes)
[Differences between Windows PowerShell 5.1 and PowerShell 7.x / Removal of the more function](https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.4#removal-of-the-more-function)

## Test Plan
**terminal**
* Powershell Version: 5.1.22621.3880 (Windows)
image
* Powershell Version: 7.4.3 (Windows)
image

## Result (Website)
image

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ uv is installed to `~/.cargo/bin`.
     curl -LsSf https://astral.sh/uv/install.sh | less
     ```
 
+    On Windows
+    ```powershell
+    powershell -c "irm https://astral.sh/uv/install.ps1 | more"
+    ```
```
Comment (zanieb) [inline docs/installation.md:30]:
```suggestion
    ```

```

**Review by zanieb — APPROVED** [review]
Thanks! Updated this to match the code block style above.

**Comment by FishAlchemist** [comment]
> Thanks! Updated this to match the code block style above.

I intentionally split them into two code blocks so that only the expected commands would be copied to the clipboard.

**Comment by zanieb** [comment]
Fair, but we need to be consistent — we can't change styles mid-section like that. We should consider changing both instead.

**Pull request merged**